Update meta descriptions in SDK docs v5 - Closes #828 (#829)
LiskHQ/lisk-docs: modules/ROOT/pages/references/lisk-framework/index.adoc

= Lisk Framework
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Framework overview covers its modules and components including the BaseModule, the BaseAsset, and the BasePlugin. 

Reviewer: ```suggestion
:description: Covers references of the Lisk Framework, like the BaseModule, the BaseAsset and the BasePlugin. 
```
